### DISCUSSION TITLE
Error 0xc000007b

### DISCUSSION THREAD

**Discussion by LughWalle** [monocasual/giada#667] [Q&A] [answered]
I downloaded version 0.25.1 for windows, the first time I had a problem with libmp3lame.dll, I solved it by downloading the dll, but now when I try to open the file it gives the error "the application cannot be started correctly (0xc000007b)"

**Comment by demontrees** [comment]
Same Issue. Does Giada not work with Win11?

**Comment by gvnnz** [comment, marked as answer]
Hey @LughWalle and @demontrees , this is a known issue that has been fixed on master branch recently (commit https://github.com/monocasual/giada/commit/89db096848d0b5a18d7d588ec0cefa24121e0649). The fix will be shipped soon in version 0.26. Thanks!
